pandas-iris/README.md

```diff
@@ -1,11 +1,14 @@
-# The `pandas-iris` Kedro starter
+# [ARCHIVED] This starter is no longer maintained.
+You can still find the starter code in older release tags: https://github.com/kedro-org/kedro-starters/tree/0.18.14/pandas-iris 
 
-## Introduction
+## The `pandas-iris` Kedro starter
+
+### Introduction
 
 The code in this repository demonstrates best practice when working with Kedro. It contains a Kedro starter template with some initial configuration and an example pipeline, and originates from the [Kedro Iris dataset example](https://docs.kedro.org/en/stable/get_started/new_project.html).
```

Comment:
```suggestion
The code in this repository demonstrates best practice when working with Kedro. It contains a Kedro starter template with some initial configuration and an example pipeline, and originates from the [Kedro Iris dataset example](https://docs.kedro.org/en/0.18.14/get_started/new_project.html).
```

Reply:
The Iris starter will be removed from docs from 0.19.0 so I've pinned to 0.18.14. We need to do this wherever we retain mention of the Iris example if we want people to read the docs about it.

Reply:
I've updated other mentions in the starters as well.